pkg/runtime/dispatcher.go

```diff
@@ -236,15 +236,21 @@ func (m *dispatcher) Quota(ctx context.Context, requestBag attribute.Bag,
 	qres, err := m.dispatch(ctx, requestBag, adptTmpl.TEMPLATE_VARIETY_QUOTA,
 		func(call *Action) []dispatchFn {
 			for _, inst := range call.instanceConfig {
-				if inst.Name != qma.Quota {
-					continue
-				}
+				// if inst.Name != qma.Quota {
```

Comment:
what is the harm in leaving this check enabled?

Reply:
It will not work. The qma.Name == `RequestCount`,  however instance names in config are fully qualified. So mixer client **will** have to send the specific fully qualified name over.